pkg/daemon/ceph/client/mirror.go

```diff
@@ -53,7 +53,7 @@ type Images struct {
 var (
 	rbdMirrorPeerCaps                     = []string{"mon", "profile rbd-mirror-peer", "osd", "profile rbd"}
 	rbdMirrorPeerKeyringID                = "rbd-mirror-peer"
-	radosNamespaceMirroringMinimumVersion = cephver.CephVersion{Major: 20, Minor: 0, Extra: 0}
+	radosNamespaceMirroringMinimumVersion = cephver.CephVersion{Major: 19, Minor: 2, Extra: 0}
```

Comment:
How about a simple comment about this change? something like this:
```suggestion
	// Modified for downstream
	radosNamespaceMirroringMinimumVersion = cephver.CephVersion{Major: 19, Minor: 2, Extra: 0}
```